doc/api/util.md

```diff
@@ -1393,6 +1393,33 @@ added: v10.0.0
 
 Returns `true` if the value is a native `External` value.
 
+```c++
+#include <js_native_api.h>
+#include <string.h>
+napi_value result;
+static napi_value MyNapi(napi_env env, napi_callback_info info) {
+  int* raw = (int*) malloc(1024);
+  napi_status status = napi_create_external(env, (void*) raw, NULL, NULL, &result);
+  if (status != napi_ok) {
+    napi_throw_error(env, NULL, "napi_create_external failed");
+    return NULL;
+  }
+  return result;
+}
+...
+DECLARE_NAPI_PROPERTY("myNapi", MyNapi)
+...
+```
+```js
```

Comment:
Nit: it is probably a good idea to add a newline here for readability purposes.

```suggestion

```js
```

Reply:
@BridgeAR, ditto.